src/ripple/basics/strHex.h

```diff
@@ -25,6 +25,7 @@
 #ifndef RIPPLE_BASICS_STRHEX_H_INCLUDED
 #define RIPPLE_BASICS_STRHEX_H_INCLUDED
 
+#include <cassert>
```

Comment:
True.  That header is required but is already included earlier in the translation unit… for now.   A subsequent commit changes that, and the missing include is now necessary.  Since it belonged there in the first place I treated it as a fix and committed it separately.